tools/releaseBuild/Images/microsoft_powershell_alpine3/build-and-run-pwsh.sh

```diff
@@ -62,13 +72,13 @@ cd ..
 targetFile="Microsoft.PowerShell.SDK/obj/Microsoft.PowerShell.SDK.csproj.TypeCatalog.targets"
 cat > $targetFile <<-"EOF"
 <Project>
-  <Target Name="_GetDependencies"
-          DependsOnTargets="ResolveAssemblyReferencesDesignTime">
-    <ItemGroup>
-      <_RefAssemblyPath Include="%(_ReferencesFromRAR.ResolvedPath)%3B" Condition=" '%(_ReferencesFromRAR.Type)' == 'assembly' And '%(_ReferencesFromRAR.PackageName)' != 'Microsoft.Management.Infrastructure' " />
-    </ItemGroup>
-    <WriteLinesToFile File="$(_DependencyFile)" Lines="@(_RefAssemblyPath)" Overwrite="true" />
-  </Target>
+    <Target Name="_GetDependencies"
```

Comment:
Minor nit: Are the white space diffs really needed?

Reply:
Consistent 4 spaces indentation (was copied from build.psm1)